packages/venia-ui/i18n/en_US.json

```diff
@@ -376,10 +378,10 @@
     "validation.isRequired": "Is required.",
     "validation.mustBeChecked": "Must be checked.",
     "validation.validatePassword": "A password must contain at least 3 of the following: lowercase, uppercase, digits, special characters.",
+    "wishlist.emptyListText": "There are currently no items in this list",
+    "wishlistButton.addedText": "Added to Favorites",
```

Comment:
Alphabetically they are correct arent they?
![image](https://user-images.githubusercontent.com/35203638/115903194-eb676100-a428-11eb-8356-61544d4c7ab5.png)

Reply:
I don't think so :P

Reply:
Honestly we should just set up a lint rule to auto sort this file for us :D

Reply:
Prefer standard string-value sorting, please, as it's the least arbitrary.

```
addError
addText
addedText
```

Reply:
Yeah, the plugin I used is not considering the length of the keys, it is only considering based on the character in the position. Since `addEr` comes after `added` it sorted it that way.